test/e2e/kubevirt.go

```diff
@@ -1698,8 +1836,10 @@ runcmd:
 						Should(Equal([]string{expectedIPv6GatewayPath}), "should filter remote ipv6 gateway nexthop")
 				}
 				step = by(vmi.Name, fmt.Sprintf("Check north/south traffic before %s %s", td.resource.description, td.test.description))
-				startNorthSouthIngressIperfTraffic(externalContainerName, nodeIPs, svc.Spec.Ports[0].NodePort, step)
-				checkNorthSouthIngressIperfTraffic(externalContainerName, nodeIPs, svc.Spec.Ports[0].NodePort, step)
+				output, err := kubevirt.RunCommand(vmi, "/tmp/iperf-server.sh &", time.Minute)
```

Comment:
Why is the script ran in background? isn't the iperf3 in it already supposed to be ran in background?

Reply:
Good catch!, I think it was a blocking script at the beginning and I forgot to remove this